PatientRegistrationApp/src/App.vue

```diff
@@ -152,36 +152,52 @@
 					<v-footer absolute color="white">
 						<template v-if="isGreetingPage()">
 							<v-spacer></v-spacer>
-							<v-icon large color="secondary" @click="goToNextPage()">
-								mdi-chevron-right
-							</v-icon>
+							<v-btn color="secondary" class="ma-2 white--text" @click="goToNextPage()">
+								Continue
+								<v-icon right large color="white">
+									mdi-chevron-right
+								</v-icon>
+							</v-btn>
 						</template>
 						<template v-else-if="isSinglePatientReviewSubmit() && isSinglePatientRegistration()">
-							<v-icon large color="secondary" @click="goToPreviousPage()">
-								mdi-chevron-left
-							</v-icon>
+							<v-btn color="secondary" class="ma-2 white--text" @click="goToPreviousPage()">
```

Comment:
We get this weird bug regardless of the options that have been selected. I'm currently in the single patient registration workflow trying clicking continue. The previous page's selections don't matter either

![image](https://user-images.githubusercontent.com/67079030/100695055-08a27280-3356-11eb-8a48-730c3f4d4d18.png)

Reply:
This same bug exists in the household workflow as well. Back button looks fine however.

Reply:
Yep!  I encountered this issue when updating from development, but Shahd and I fixed that with this merge:
![image](https://user-images.githubusercontent.com/61951196/100782196-b141e800-33d1-11eb-8bbd-663b759cb1e1.png)
After updating from development again, this is resolved.